deps_licenses/licenses_linux_user.txt

```diff
@@ -32,35 +32,35 @@ nvidia-curand-cu12, 10.3.2.106, Other/Proprietary License
 nvidia-cusolver-cu12, 11.4.5.107, Other/Proprietary License
 nvidia-cusparse-cu12, 12.1.0.106, Other/Proprietary License
 nvidia-nccl-cu12, 2.20.5, Other/Proprietary License
-nvidia-nvjitlink-cu12, 12.5.82, Other/Proprietary License
+nvidia-nvjitlink-cu12, 12.6.68, Other/Proprietary License
 nvidia-nvtx-cu12, 12.1.105, Other/Proprietary License
 onnx, 1.16.1, Apache License v2.0
 onnxconverter-common, 1.13.0, MIT License
-onnxmltools, 1.11.0, Apache Software License
+onnxmltools, 1.12.0, Apache Software License
 onnxoptimizer, 0.3.13, Apache License v2.0
 onnxruntime, 1.18.0, MIT License
 packaging, 24.1, Apache Software License; BSD License
 pandas, 2.0.3, BSD License
 protobuf, 4.25.5, 3-Clause BSD License
 psutil, 5.9.8, BSD License
 python-dateutil, 2.9.0.post0, Apache Software License; BSD License
-pytz, 2024.1, MIT License
+pytz, 2024.2, MIT License
 requests, 2.32.3, Apache Software License
 scikit-learn, 1.1.3, BSD License
 scipy, 1.10.1, BSD License
 six, 1.16.0, MIT License
-skl2onnx, 1.12, Apache Software License
+skl2onnx, 1.17.0, Apache Software License
```

Comment:
did you poetry lock run with the no-update? I am surprised there are so many dep changes

Reply:
thanks for the reminder, i forgot the --no update